src/applications/vaos/tests/e2e/appointment-list.cypress.spec.js

```diff
@@ -66,7 +66,7 @@ describe('VAOS appointment list refresh', () => {
 
   describe('appointments details', () => {
     beforeEach(() => {
-      cy.findByText(/Your appointments/i).should('exist');
```

Comment:
why would cypress not detect "Your appointments" for the appointment list page?

Reply:
I ran the tests locally after reverting this line and it looks like it's failing because it's detecting multiple instances of `Your appointments` and not because it can't find it. The `findByText` query seems to include the document title. 

I'm ok with the new method but we could do something like `cy.get('h2').contains(/Your appointments/i);` as well.

Reply:
Actually, now that each appointment type page has a distinct title maybe it makes sense to query the `h2` specifically in each describe block.